src/test/test_bitcoin.cpp

```diff
@@ -25,7 +25,7 @@
 
 CClientUIInterface uiInterface; // Declared but not defined in ui_interface.h
 CWallet* pwalletMain;
-libzerocash::ZerocashParams *pzerocashParams;
+ZCJoinSplit *pzcashParams;
```

Comment:
Nah, it's just exported from `libbitcoinserver` (or whatever it is) just as `pwalletMain` is.